### DISCUSSION TITLE
How would I go about checking when last a list was used?

### DISCUSSION THREAD

**Discussion by phippsac** [sympa-community/sympa#1714] [Q&A] [answered]
Hi there I want to check when last all the lists in Sympa were used? I know I can get this information from the web interface but I have > 1000 lists so would like to be able to extract the information from the DB, in particular the logs_table.

So basically I need the name of the list and when it last had activity on it and would like to run it against all my current lists. 

Apologies my mysql knowledge is basic to say the least. 

Any help would be appreciated.

Thanks

**Comment by ikedas** [comment, marked as answer]
The time of the last activity is not stored in the database.

The easiest way to get such a time would be to get a timestamp of `stats` or `msg_count` file in [each list directory](https://www.sympa.community/manual/customize/basics-list-config.html). This will match the last time a message was delivered through the list.

**Reply by phippsac** [reply]
Thanks so much for pointing me in the right direction, much appreciated.

**Reply by phippsac** [reply]
That didn't work for me unfortunately as I regenerated the owner.dump files recently,

**Reply by ikedas** [reply]
The timestamp of the `owner.dump` file doesn't matter.

**Reply by phippsac** [reply]
Hi there. When regenerating the new owner.dump files,  the stats and msg_count files were accessed and modified so that data is not helpful for checking last activity, this is in my scenario. Maybe different for other environments. Any other suggestions? Thanks

**Reply by ikedas** [reply]
> When regenerating the new owner.dump files, the stats and msg_count files were accessed and modified...
Could you please show the results of `ls -l` before and after you regenerated `owner.dump`?

**Comment by phippsac** [comment]
I was able to extract logs information from the logs_table in the db thanks.
